src/helm/benchmark/run.py

```diff
@@ -256,6 +263,8 @@ def main():
         register_huggingface_hub_model_config(huggingface_model_name)
     for huggingface_model_path in args.enable_local_huggingface_models:
         register_huggingface_local_model_config(huggingface_model_path)
+    for model_config_path in args.model_config_paths:
+        register_model_configs_from_path(model_config_path)
```

Comment:
Does this mechanism subsume the Hugging Face local model loading?

Reply:
Yes, eventually this will replace the Hugging Face mechanism (after some additional refactoring).